.github/workflows/scheduled-production-snapshot.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: Production Media Snapshot
+on:
+  #schedule:
+    # Invoke at 9 UTC every monday
+    #- cron: '0 9 * * 1'
```

Comment:
Are all of these production workflows' schedules commented about because we want to test in lower environments first?